<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSelect</class>
 <widget class="QDialog" name="CDlgSelect">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LGCK Builder - choose</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lgck-builder.qrc">
    <normaloff>:/images/lgckbuilder64ronnie2.png</normaloff>:/images/lgckbuilder64ronnie2.png</iconset>
  </property>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>368</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open an existing game project.</string>
   </property>
   <property name="whatsThis">
    <string>This is the button to open an existing game.</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/splash/open.png</normaloff>:/images/splash/open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>368</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Create a new game project.</string>
   </property>
   <property name="whatsThis">
    <string>This is the button to create a new game.</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/splash/new.png</normaloff>:/images/splash/new.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSkip">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>470</y>
     <width>441</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This is the button to skip this splash screen.</string>
   </property>
   <property name="whatsThis">
    <string>This is the button to skip this splash screen.</string>
   </property>
   <property name="text">
    <string>Skip this Splash Screen &gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>630</width>
     <height>461</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="lgck-builder.qrc">:/images/Frank2_scaled33.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNoShow">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>470</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This is the button to click if don't want to use the splash screen.</string>
   </property>
   <property name="whatsThis">
    <string>This is the button to click if don't want to use the splash screen.</string>
   </property>
   <property name="text">
    <string>  Do not show again</string>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/pd/dagobert83_cancelx16.png</normaloff>:/images/pd/dagobert83_cancelx16.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>13</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSpriteEditor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>368</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Launch the Sprite Editor</string>
   </property>
   <property name="whatsThis">
    <string>This is the button to lauch the sprite editor.</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/splash/editor.png</normaloff>:/images/splash/editor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTutorials">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>368</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Watch tutorials on how to make games.</string>
   </property>
   <property name="whatsThis">
    <string>Watch tutorials on how to make games.</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="lgck-builder.qrc">
     <normaloff>:/images/splash/tutorials.png</normaloff>:/images/splash/tutorials.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>90</height>
    </size>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnOpen</zorder>
  <zorder>btnNew</zorder>
  <zorder>btnSkip</zorder>
  <zorder>btnNoShow</zorder>
  <zorder>btnSpriteEditor</zorder>
  <zorder>btnTutorials</zorder>
 </widget>
 <resources>
  <include location="lgck-builder.qrc"/>
 </resources>
 <connections/>
</ui>
